pkg/logger/worker.go

```diff
@@ -55,7 +55,7 @@ func (W *Worker) sendCloudEvent(logReq LogRequest) error {
 
 	t, err := cloudevents.NewHTTPTransport(
 		cloudevents.WithTarget(logReq.Url.String()),
-		cloudevents.WithEncoding(cloudevents.HTTPBinaryV1),
+		cloudevents.WithEncoding(cloudevents.HTTPBinaryV01),
```

Comment:
Why the change back to 01 when V1 is the released version?

Reply:
we are using cloudevent sdk-go 1.2.0 which does not have HTTPBinaryV1 and caused test failure, which version of sdk-go we should use?

Reply:
hmm actually I have some old vendor lingering around, there is `HTTPBinaryV1` but I still get this error
```
{"level":"error","ts":1594634991.9612372,"logger":"entrypoint","msg":"Failed to send log","URL":"http://message-dumper-predictor-default.kfserving-ci-e2e-test","error":"while sending event: error sending cloudevent: 400 Bad Request","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\tgithub.com/go-logr/zapr@v0.1.1/zapr.go:128\ngithub.com/kubeflow/kfserving/pkg/logger.(*Worker).Start.func1\n\tgithub.com/kubeflow/kfserving/pkg/logger/worker.go:109"}
```

Reply:
it works with `HTTPBinaryV01` though, I am now confused.

Reply:
ok confirmed that the issue is caused by the stale event display image which should be `gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display` and we still should us `HTTPBinaryV1`